client/app/telecom/task/telecom-task.controller.js

```diff
@@ -18,6 +18,7 @@ angular.module("managerApp").controller("TelecomTaskCtrl", function ($scope, $tr
         },
         tip: $translate.instant("task_all")
     };
+    ouiDatagridConfiguration.translations.emptyPlaceholder = $translate.instant("telecom_task_none");
```

Comment:
Do not change global configuration like that. This affects other pages. There no way to define a specific empty placeholder for a datagrid for now.